packages/jest-config/src/constants.ts

```diff
@@ -8,8 +8,8 @@
 import * as path from 'path';
 
 export const NODE_MODULES = `${path.sep}node_modules${path.sep}`;
+export const BUILD_IN_REPORTERS = ['default', 'github-actions'];
```

Comment:
`BUILT`? Not sure, it's missing coverage reporter, etc

Reply:
I am not a fan of this too. No need to import it in `TestScheduler`. If one day `@jest/test-scheduler` will be separate package, it will be redundant to have `jest-config` as a dependency. Perhaps should be simply inlined?